specification/apicenter/ApiCenter.Management/examples/2024-06-01-preview/ApiSources_CreateOrUpdate.json

```diff
@@ -0,0 +1,59 @@
+{
+  "operationId": "ApiSources_CreateOrUpdate",
+  "title": "ApiSources_CreateOrUpdate",
+  "parameters": {
+    "resourceGroupName": "contoso-resources",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "serviceName": "contoso",
+    "workspaceName": "default",
+    "apiSourceName": "contoso-api-management",
+    "api-version": "2024-06-01-preview",
+    "resource": {
+      "properties": {
+        "azureApiManagementSource": {
+          "resourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-resources/providers/Microsoft.ApiManagement/service/contoso",
+          "msiResourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-resources/providers/Microsoft.ManagedIdentity/userAssignedIdentities/contoso-identity"
+        },
+        "importSpecification": "ondemand"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-resources/providers/Microsoft.ApiCenter/services/contoso/workspaces/default/apiSources/contoso-api-management",
+        "type": "Microsoft.ApiCenter/services/workspaces/apiSources",
+        "name": "contoso-api-management",
+        "systemData": {
+          "createdAt": "2023-07-03T18:27:09.128871Z",
+          "lastModifiedAt": "2023-07-03T18:27:09.1288716Z"
+        },
+        "properties": {
+          "azureApiManagementSource": {
+            "resourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-resources/providers/Microsoft.ApiManagement/service/contoso",
+            "msiResourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-resources/providers/Microsoft.ManagedIdentity/userAssignedIdentities/contoso-identity"
+          },
+          "importSpecification": "ondemand"
+        }
+      }
+    },
+    "201": {
+      "body": {
+        "id": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-resources/providers/Microsoft.ApiCenter/services/contoso/workspaces/default/apiSources/contoso-api-management",
+        "type": "Microsoft.ApiCenter/services/workspaces/apiSources",
+        "name": "contoso-api-management",
+        "systemData": {
+          "createdAt": "2023-07-03T18:27:09.128871Z",
+          "lastModifiedAt": "2023-07-03T18:27:09.1288716Z"
+        },
+        "properties": {
+          "azureApiManagementSource": {
+            "resourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-resources/providers/Microsoft.ApiManagement/service/contoso",
+            "msiResourceId": "/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/contoso-resources/providers/Microsoft.ManagedIdentity/userAssignedIdentities/contoso-identity"
+          },
+          "importSpecification": "ondemand"
```

Comment:
Should we update these examples with the other API source properties e.g `targetLifecycleStage` and `targetEnvironmentId`?

Reply:
Added to examples.